catalog/package.json

```diff
@@ -16,7 +16,7 @@
     "analyze:clean": "rimraf stats.json",
     "preanalyze": "npm run analyze:clean",
     "analyze": "cross-env NODE_ENV=production node ./internals/scripts/analyze.js",
-    "bundlesize": "bundlesize -f './build/*.js'",
+    "bundlewatch": "bundlewatch './build/*.js'",
```

Comment:
Does It use the default size limit? I can't find its configurating

Reply:
I see you comment for Aneesh

Reply:
it doesnt use any size limits, just tracking the changes (at least it should start doing so after we merge it)